index.html

```diff
@@ -280,5 +280,19 @@ <h5>Skills</h5>
 				</div>
 			</div>
 		</section>
+		<section id="contact_form">
+			<div class="contact_detail">
+				<h2 class="head">Contact me</h2>
+				<p class="detail">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+					I’d love to help with it!
+				</p>
+			</div>
+			<form action="https://formspree.io/f/mknalvyr" method="POST" class="form">
+					<input class="input" type="text" name="text" maxlength="30" required placeholder="Enter your name">
+					<input class="input" type="email" name="email" required placeholder="Enter your email">
+					<textarea class="input" name="message" placeholder="Enter message" rows="5" required></textarea>
```

Comment:
- [ ] Kindly 😃 It required that the textarea field accepts up to 500 chars, therefore kindly add the maxlength="500" attribute to the name field to stick to the requirements .💯👌